starlark/unpack.go

```diff
@@ -10,6 +10,12 @@ import (
 	"strings"
 )
 
+// Unpacker defines an interface for custom types that implement
+// their own unpacking logic.
+type Unpacker interface {
+	Unpack(v Value) error
+}
+
```

Comment:
Update the UnpackArgs comment:

```
// it and all following parameters are optional.
//
// If the variable implements Unpacker, its Unpack argument
// is called with the argument value, allowing an application
// to define its own argument validation and conversion.
//
// If the variable implements Value...
```

(Why won't GitHub let me comment on faraway lines? Grrr.)

Reply:
updated!